react/src/base/inputs/DatePicker.stories.js

```diff
@@ -33,30 +36,49 @@ props table below for available customization options.
 
 export const defaultStory = () => (
   <SprkInputContainer>
-    <SprkDatePickerInput
-      name="date"
-      placeholder="01/01/2019"
-      errorMessage="Incorrect date."
-      label="Date"
-    />
+    <div className="sprk-b-InputContainer__icon-container">
```

Comment:
can this div be part of SprkInputContainer? not sure if it's worth it but I'd like to bring it up

Reply:
This was intentional to keep all the "is there/isn't there an icon" checks out of the SprkInputContainer and keep it more compositional. This is in line with how we did the Angular input API upgrades.